### PR TITLE
Chore: some type fixes

### PR DESCRIPTION
**Pull request by ashharrison90**
**What is this feature?**

- fixing `any`s/type assertions 🖖 

**Why do we need this feature?**

- better typings across grafana

**Who is this feature for?**

- grafana devs

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: public/app/plugins/panel/debug/EventBusLogger.tsx [hunk 1]
```diff
@@ -52,7 +52,7 @@ export class EventBusLoggerPanel extends PureComponent<Props, State> {
 
   eventObserver: PartialObserver<BusEvent> = {
     next: (event: BusEvent) => {
-      const origin = event.origin as any;
+      const origin: any = event.origin;
```
Comment (joshhunt) [inline public/app/plugins/panel/debug/EventBusLogger.tsx:55]: lol i guess

File: public/app/plugins/panel/nodeGraph/usePanning.ts [hunk 2]
```diff
@@ -184,10 +184,10 @@ function inBounds(value: number, min: number | undefined, max: number | undefine
 }
 
 // The issue here is that TouchEvent is undefined while using instanceof in Firefox and Safari
-// which will throw an exception but if it's (event as TouchEvent).changedTouches it will be undefined
+// which will throw an exception but if it's event.changedTouches it will be undefined
```
Comment (joshhunt) [inline public/app/plugins/panel/nodeGraph/usePanning.ts:187]: lol

**Review by joshhunt — APPROVED** [review]

File: public/app/plugins/panel/debug/EventBusLogger.tsx (hunk 1 above)
Comment (ashharrison90) [reply public/app/plugins/panel/debug/EventBusLogger.tsx:55]: i _know_, but number still needs to go down before i can face actually tackling problems 😂

**Review by fabrizio-grafana — APPROVED** [review]

**Review by joey-grafana — APPROVED** [review]
NodeGraph looks good 👍

**Pull request merged**